packages/babel-preset-env/test/fixtures/plugins-integration/issue-7527/exec.js

```diff
@@ -0,0 +1,8 @@
+class MyDate extends Date {
+  constructor(time) {
+      super(time);
+  }
+}
+let myDate = new MyDate();
+
+assert.equal(myDate.toString, Date.prototype.toString);
```

Comment:
This is not equivalent to my proposed test. The method has to be executed, because their an internal branding check (that’s the actual test, not the `typeof`).

Reply:
I think you need both.